### PR TITLE
Added new compression method

### PR DESCRIPTION
**Pull request by DKaramfilov**
Added new compression method

### CODE REVIEW

File: libraries/radpdfprocessing/formats-and-conversion/pdf/pdfformatprovider/settings.md [hunk 1]
```diff
@@ -178,9 +178,15 @@ __Example 4__ shows how you can create a __PdfExportSettings__ object with setti
...
 provider.ExportSettings = settings;
 
-
 {{endregion}}
 
+### __StreamCompression__
+
+ Gets or sets the content stream compression type. Possible Values are: 
+ * **None**: The content streams won't be encoded.
+ * **FlateDecode**:  Compresses data using the zlib/deflate compression method.
```
Comment (YoanKar) [inline libraries/radpdfprocessing/formats-and-conversion/pdf/pdfformatprovider/settings.md:187]: Double space before "Compresses".

File: libraries/radpdfprocessing/formats-and-conversion/pdf/pdfstreamwriter/pdfstreamwriter.md [hunk 2]
```diff
@@ -95,6 +95,10 @@ Through the **Settings property** of PdfStreamWriter you can control the way the
 
 * **WriteAnnotations**: A boolean property indicating whether the annotations should be included in the exported document.
 
+* **StreamCompression**: Gets or sets the content stream compression type. Possible Values are: 
+   - **None**: The content streams won't be encoded.
+   - **FlateDecode**:  Compresses data using the zlib/deflate compression method.
```
Comment (YoanKar) [inline libraries/radpdfprocessing/formats-and-conversion/pdf/pdfstreamwriter/pdfstreamwriter.md:100]: Double space before "Compresses".

File: libraries/radpdfprocessing/formats-and-conversion/pdf/pdfformatprovider/settings.md (hunk 1 above)
Comment (DKaramfilov) [reply libraries/radpdfprocessing/formats-and-conversion/pdf/pdfformatprovider/settings.md]: fixed

File: libraries/radpdfprocessing/formats-and-conversion/pdf/pdfstreamwriter/pdfstreamwriter.md (hunk 2 above)
Comment (DKaramfilov) [reply libraries/radpdfprocessing/formats-and-conversion/pdf/pdfstreamwriter/pdfstreamwriter.md]: fixed

**Pull request merged**